### PR TITLE
grab chalk and use it to make the build and engine loading process pr…

### PR DESCRIPTION
**Pull request by geoffp**
I wanted it to be pretty, so I did stuff to it.
screenshot from 2016-11-23 23-12-03

### CODE REVIEW

**Comment by raphaelokon** [comment]
I switched switched CLI from `colors` to `chalk` the other day as well. In future would be good to log verbosely via a CLI flag … I use a small event-driven logger for that in the CLI …

https://github.com/pattern-lab/patternlab-node-cli/blob/master/bin/utils.js#L10-L25

**Comment by tburny** [comment]
I'd like to suggest replacing all console.log statements with more robust logging - with no exceptions.
This way logs become parsable by standard command line tools like cut, sed/awk, etc. We get some timings for free, to see which step takes how long.

In summary I like the idea :)

**Comment by geoffp** [comment]
@raphaelokon, that looks nice, but what's the benefit of using an event bus for logging, as opposed to just calling a simple logging function directly?

@tburny, are you after some kind of delimiter that a parser can grab onto, or timestamps for every log entry, or what? Gulp does something that, and I like that it tells you how long tasks take.

**Comment by raphaelokon** [comment]
@geoffp The main benefit is that the event listener can decide what to do with the logs that were emitted. I do not have to log them in place, but can decide how to pipe them into different listeners (eg. writing log files, console.log in specific format etc). 

@tburny open to formats here.

**Comment by raphaelokon** [comment]
@geoffp `console.log` is also blocking on tty terminals, like OS X due to limited buffer size (I think 1KB or so)

**Comment by bmuenzenmeyer** [comment]
I've endeavored to keep dependencies low - hence the current pretty ghetto implementation. I'd like feedback from users regarding additional, perhaps cosmetic dependencies.

I also like the idea @raphaelokon proposed about a better logger. Seems like a more robust way to emit and listen to events of differening importance to the user / use case.

**Comment by bmuenzenmeyer** [comment]
I've come around to this - I created #566 to finish it up.
Another issue / pr could be to replace `console.log` with something better across the board.

File: core/lib/patternlab.js [hunk 1]
```diff
@@ -14,13 +14,21 @@ var diveSync = require('diveSync'),
...
   pm = require('./plugin_manager'),
   fs = require('fs-extra'),
-  plutils = require('./utilities'),
-  patternEngines = require('./pattern_engines');
+  packageInfo = require('../../package.json'),
+  plutils = require('./utilities');
 
+console.log(
+  chalk.bold('\n====[ Pattern Lab / Node'),
+  `- v${packageInfo.version}`,
```
Comment (bmuenzenmeyer) [inline core/lib/patternlab.js]: I really like what this adds to output Geoff

**Comment by bmuenzenmeyer** [comment]
> @geoffp console.log is also blocking on tty terminals, like OS X due to limited buffer size (I think 1KB or so)

@raphaelokon would you recommend utilizing your event logger in this case then?

**Comment by raphaelokon** [comment]
The event-based logger just delegates the logging part to the handler function(s) you call … if you call `console.log` in a handler, it would also block afaik.

**Pull request merged**